spec/rubocop/cop/i18n/gettext/decorate_string_formatting_using_interpolation_spec.rb

```diff
@@ -1,18 +1,18 @@
 # frozen_string_literal: true
+
 require 'spec_helper'
 
 describe RuboCop::Cop::I18n::GetText::DecorateStringFormattingUsingInterpolation do
-  let(:config) {RuboCop::Config.new}
-  subject(:cop) {described_class.new(config)}
-  before(:each) {
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+  before(:each) do
     investigate(cop, source)
-  }
+  end
 
-  decorators = ['_', 'n_', 'N_']
+  decorators = %w[_ n_ N_]
   decorators.each do |decorator|
-
     context "#{decorator} decoration not used" do
-      it_behaves_like 'a_no_cop_required', "thing(\"a #{true} that is not decorated\")"
+      it_behaves_like 'a_no_cop_required', 'thing("a true that is not decorated")'
```

Comment:
Actually this was verifying that a method name 'thing' that wasn't part of the supported list and had a #{} in a string, wouldn't trigger the cop.

Reply:
Huh, that's an odd thing for the linter to auto-fix. I guess because the # wasn't escaped, it did the auto-fix. I'll switch it back and see what rubocop complains about.